docs/fidesops/docs/guides/oauth.md

```diff
@@ -2,147 +2,92 @@
 
-Set this bearer token to create other resources in fidesops, by including an `Authorization: Bearer <ACCESS_TOKEN>` header. 
+```
+curl \
+  -X POST 'http://<HOST>:8080/api/v1/oauth/client' \
+  -H 'Authorization: Bearer <root_access_token>'
+  -H 'Content-Type: application/json'
+  -d '{ "scopes": ["policy:read", "rule:read"]}'
+```
 
+For this call, we have to populate the `Authorization` header. Notice that the header's value is formed  as `Bearer <token>`. We also have to declare the request's `Content-Type` to be `application/json`.
+### Authorize your Client with Scopes
 
-## Example Auth Flow
+To add scopes to the client, the body of your request must contain an array of scope tokens. 
 
-After having configured the root client id and secret, I wish to use the root client to create an additional client.
+You can retrieve the available scopes by calling [`GET /api/v1/oauth/scopes`](/api#operations-OAuth-read_scopes_api_v1_oauth_scope_get), or you can look in [the scope registry file](https://github.com/ethyca/solon/blob/main/src/fidesops/api/v1/scope_registry.py).
 
-First, I'll need to obtain my root access token:
+If the call is successful, Fidesapi responds with a new client ID/client secret pair:
 
-```bash
-curl -X 'POST' \
-  'http://0.0.0.0:8080/api/v1/oauth/token' \
-  -H 'accept: application/json' \
-  -H 'Content-Type: application/x-www-form-urlencoded' \
-  -d 'grant_type=&scope=&client_id=<CLIENT_ID>&client_secret=<CLIENT_SECRET>'
 ```
+HTTP/1.1 200 OK
+Content-Type: application/json
 
-The response will look something like this:
-```bash
 {
-  "access_token": "eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..5e3CF5Y5QglasdhNMiadshFAQ.OQF47Y_IuPoSH3HD5_Ybnd5rAx8uoj3zbKGRu-mfiPrJOzUocpJh64AXKIGLinrSxv_a2Jtr6sMkD5w6kGAbHUu1ytD17Z2V66uC5WA5TJZoI8fEcsFeq6NHt-PDIXnokJkwgDuXD4nsUDH1DyP6xN4iJcBE2yF_cNvJi00WMO39Mk5gS3JhDCgydkErYjiGy3uafZNawxgyBjBttigJRimgkC3J12Sy7luPLjM9gEFS0jmg0KuwZ9f-ebYSHPy7gfyMn03CP90l_W9zAQuNUXC4EfJoMdbe4iaqjic83UXxznKTFvN8CWuzEM32Kz4O9sSUOZBKRPgZ6gwfD3Lr0VOYpuMGWNX4l6lAqUa8xM-dygVOZ1Yt-_pGmsEgQrYInQmFnDj2uij-7gcUyP_HLww0_Mi0U_yzQ67FwntFIn3vNZLO3NSf7ydWQ3aLadQNuJppj-5Q_4xewfYOPiOEBzLI3Jd4LG1poK91yWYRbfV7M6AtSBPlnToCfs58_APU7ngqLyIlrWSQxuNx-FbHKAb5pi1YRfZpUgFeKMQZeVIvbEstaHX3c9j_pQbs255VJSHG22nCP2zoKYtt5C1Mg5o617Z92uhsReapBYbponwWN7Dqiy4-3WyZ6dfhk8euJOZ0ZFlHiey48NZxe4oNBk2tmsBvFrhdLIjKeNwVXhKwS3LPnAdq3Resu6yIHl3KEHp5zuErruOCzmQxZhQcuXPJq7onpA.EcAX36Pmi2lbEM4Xhd9Xlg"
+  "client_id" : "<new_client_id>"
+  "client_secret" : "<new_client_secret>",
 }
 ```
+## Create an Access Token
+You then create a new access token by calling [`POST /api/v1/oauth/token`](/api#operations-OAuth-acquire_access_token_api_v1_oauth_token_post) with the new credentials. 
 
-Next, I can use that access token to create the new client:
+In the above example, your new access token only lets the client read policies and rules -- the client nor create other clients, nor write policies, nor perform other operations using Fidesops APIs.
```

Comment:
"— the client nor create ..." should read "— the client cannot create other clients, nor..."